Agents/FenlandTrajectoryAgent/agent/exposure_calculator/count/get_greenspace_area.sparql

```diff
@@ -1,9 +1,12 @@
 PREFIX gs: <https://www.theworldavatar.com/kg/ontogreenspace/>
 PREFIX geo: <http://www.opengis.net/ont/geosparql#>
-SELECT ?feature ?function ?geometry
+PREFIX ex: <http://example.org/ns#>
```

Comment:
Surely this is not an appropriate namespace for `hasWKB`?

Reply:
refer https://github.com/cambridge-cares/TheWorldAvatar/pull/1455#discussion_r2067751424